pkg/products/observability/reconciler_test.go

```diff
@@ -273,7 +273,7 @@ func TestReconciler_reconcileComponents(t *testing.T) {
 				extraParams:   tt.fields.extraParams,
 				recorder:      tt.fields.recorder,
 			}
-			got, err := r.reconcileComponents(tt.args.ctx, tt.args.serverClient, defaultInstallationNamespace)
+			got, err := r.reconcileComponents(tt.args.ctx, tt.args.serverClient, defaultInstallationNamespace, "redhat-rhoam-")
```

Comment:
Guess this is going to be addressed as part of e2e refactor but can we use nsPrefix here as well?